python/sunbeam/properties2.py

```diff
@@ -0,0 +1,8 @@
+from os.path import isfile
```

Comment:
The name `properties2` was really weird. To be able to facilitate dynamic loading of keywords you need to expose a `Parser` class also in Python - not just hook directly into the library.